### ISSUE TITLE
p-inputNumber: maxFractionDigits and suffix doesn't work together when minFractionDigits is not set or is 0

### ISSUE THREAD

**Issue by szabogerArgon** [primefaces/primeng#11630]
### Describe the bug

When using a p-inputNumber component with maxFractionDigits, and suffix set,  if minFractionDigits is not set, or it is set to 0, after entering a decimal point (, or . depending on the locale) the user can't input a number, the suffix gets duplicated, and upon further input the field gets cleared:

    <p-inputNumber
      mode="decimal"
      [minFractionDigits]="0"
      [maxFractionDigits]="5"
      suffix="km"
    >
    </p-inputNumber>
      
![image](https://user-images.githubusercontent.com/60918069/174074676-7b09141f-0bbc-4ae6-9f51-15b8ff36d180.png)
![image](https://user-images.githubusercontent.com/60918069/174074701-bb750c14-e52c-47f4-a58d-310b83f7881c.png)
![image](https://user-images.githubusercontent.com/60918069/174074731-ccbd2ac4-8e2b-495f-856e-16bce92ccff5.png)

### Environment

Easily reproducable using the stackBlitz link provided on https://www.primefaces.org/primeng/inputnumber

### Reproducer

https://stackblitz.com/edit/primeng-inputnumber-demo-9n2xsf?file=src%2Fapp%2Fapp.component.html

### Angular version

13.0.3

### PrimeNG version

13.0.4

### Build / Runtime

Angular CLI App

### Language

ALL

### Node version (for AoT issues node --version)

14.15.0

### Steps to reproduce the behavior

1. Open reproducer link.
2. Enter 5
3. Enter .
4. Enter 5

### Expected behavior

After step 3, the input should be "5.km", not "5km.km".

**Comment by neldrak** [comment]
Same here

**Comment by squidwardthetentacles** [comment]
Had the same issue, temporary fix:

set `[minFractionDigits]="1"`
![image](https://user-images.githubusercontent.com/15376332/199656608-7442c28a-c5d4-4a62-bb86-08739e6fb8e1.png)

output:
![image](https://user-images.githubusercontent.com/15376332/199656497-6662f554-2a12-4830-998d-d0b727d041f1.png)

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,

**Comment by tommskar** [comment]
This problem appears to be fixed in PrimeNG 14.2.1 (with Angular 14.2.10).

**Comment by neldrak** [comment]
It got even worse...
